package/readme.txt

```diff
@@ -37,6 +37,7 @@ New features:
 
 Improvements:
 - When sorting a file table by filenames, the sort operation performs a locale-sensitive String comparison.
+- More search options to Editor/Viewer, synchronize search options from File Search and set caret on the first occurrence
```

Comment:
it's better not to cover everything but make it more clear - without listing the search options I don't think it is worth adding the `More search options to Editor/Viewer` part, and without explaining what it means to synchronize the search options I don't think that adding the `synchronize search options from File Search` is worth it.

maybe we can change this to something like: When viewing/editing a file that is returned by file search with text search using the internal viewer/editor, the caret is set on first occurrence of the searched text?